python/sglang/srt/server.py

```diff
@@ -168,6 +168,16 @@ async def flush_cache():
     )
 
 
+def start_profile_sync():
```

Comment:
remove `_sync` in the name. If it conflicts with the name below, change the names of the fastapi functions below.

Reply:
done